sydent/util/emailutils.py

```diff
@@ -64,16 +64,29 @@ def sendEmail(
     )
 
     allSubstitutions = {}
+
     for k, v in substitutions.items():
         allSubstitutions[k] = v
-        allSubstitutions[k + "_forhtml"] = escape(v)
-        allSubstitutions[k + "_forurl"] = urllib.parse.quote(v)
+        if not templateFile.endswith(".j2"):
+            allSubstitutions[k + "_forhtml"] = escape(v)
+            allSubstitutions[k + "_forurl"] = urllib.parse.quote(v)
```

Comment:
I think it would be clearer to move this logic into the if-block of rendering a non-Jinja template.